src/reconcile/rec_track.c

```diff
@@ -473,7 +473,7 @@ __wt_ovfl_reuse_search(WT_SESSION_IMPL *session, WT_PAGE *page,
 
 	if (WT_VERBOSE_ISSET(session, WT_VERB_OVERFLOW))
 		WT_RET(__ovfl_reuse_verbose(session, page, reuse, "reclaim"));
-	return (1);
+	return (0);
```

Comment:
Then if any function called by `__ovfl_reuse_verbose` were ever to return `WT_NOTFOUND` for any reason, we'd have reintroduced the same bug this change is supposed to fix.

I could add `int *foundp` to the `__ovfl_reuse_verbose` signature if you'd prefer something more explicit?